tests/testbitmap.c

```diff
@@ -758,6 +758,25 @@ static void test_createBitmapFromGraphics ()
 	GdipDeleteGraphics (graphicsWithResolution);
 }
 
+static void test_cloneBitmap()
+{
+	GpStatus status;
+	GpBitmap *bitmap;
+	GpBitmap *clone;
+	
+	status = GdipCreateBitmapFromScan0(1, 1, 0, PixelFormat32bppARGB, NULL, &bitmap);
+	assertEqualInt(status, Ok);
+
+	status = GdipCloneBitmapAreaI(0, 0, 1, 1, PixelFormat4bppIndexed, bitmap, &clone);
+	assertEqualInt(status, Ok);
+
+	PixelFormat actual;
+	status = GdipGetImagePixelFormat(clone, &actual);
+	assertEqualInt(status, Ok);
+
```

Comment:
I assume it would make sense to test the actual contents of the bitmap here. You can create the 32bpp bitmap from some source data like `{ 0xFFFFFFFF, 0, 0, 0xFFFFFFFF }` with size `2, 2` and then verify the pixel colors on the output (already done on some other tests).

Reply:
Thanks for the hint. Will update the test.